src/main/java/crazypants/enderio/TileEntityEio.java

```diff
@@ -32,14 +33,19 @@ protected boolean shouldUpdate() {
     return true;
   }
 
+  private long lastUpdate = 0;
+
   @Override
   public final void updateEntity() {
-    doUpdate();
-    if(isProgressTile && !worldObj.isRemote) {
-      int curScaled = getProgressScaled(16);
-      if(++ticksSinceLastProgressUpdate >= getProgressUpdateFreq() || curScaled != lastProgressScaled) {
-        sendTaskProgressPacket();
-        lastProgressScaled = curScaled;
+    if (Config.allowExternalTickSpeedup || worldObj == null || worldObj.getTotalWorldTime() != lastUpdate) {
```

Comment:
OR if `worldObj == null`? This means this is getting called if the world object is null.....